kubernetes/start

```diff
@@ -8,8 +8,26 @@ for template in $(ls manifest-templates)
 do
     log "Processing template $template..."
     sed -e "s#\${project_dir}#$(dirname $PWD)#" manifest-templates/$template > manifests/$template
+
+
+    # Use SALT_DIR is set and not empty else use our default directory
+    # This can be used to point to different state files for orchestration.
+    if [ -z ${SALT_DIR+x} ]
+    then
+      log "You did not provide a SALT_DIR. $(dirname $PWD)/kubernetes/salt will be used..."
+      salt_dir="$(dirname $PWD)/kubernetes/salt"
+      sleep 5
+    else
+      salt_dir=$SALT_DIR
+    fi
+
+    sed -i -e "s#\${salt_dir}#$salt_dir#" manifests/$template
 done
 
-log "Launching kubelet"
+./cleanup
```

Comment:
I'd prefer if you'd ask the user first before cleaning up. If the user doesn't want to do it, then we `exit 0`.